apps/authentication/backends/oauth2/backends.py

```diff
@@ -121,22 +116,12 @@ def authenticate(self, request, code=None, **kwargs):
             logger.error(log_prompt.format(error))
             return None
 
-        query_dict = self.get_query_dict(response_data, query_dict)
-
         headers = {
             'Accept': 'application/json',
             'Authorization': 'Bearer {}'.format(response_data.get('access_token', ''))
         }
-
         logger.debug(log_prompt.format('Get userinfo endpoint'))
-        if '?' in settings.AUTH_OAUTH2_PROVIDER_USERINFO_ENDPOINT:
-            separator = '&'
-        else:
-            separator = '?'
-        userinfo_url = '{url}{separator}{query}'.format(
-            url=settings.AUTH_OAUTH2_PROVIDER_USERINFO_ENDPOINT, separator=separator,
-            query=urlencode(query_dict)
-        )
+        userinfo_url = settings.AUTH_OAUTH2_PROVIDER_USERINFO_ENDPOINT
         userinfo_response = requests.get(userinfo_url, headers=headers)
         try:
             userinfo_response.raise_for_status()
```

Comment:
The code has no significant issues or oddities; it appears to be well-written with only a couple of minor improvements:

1. The class `authenticate` could benefit from documentation strings (e.g., `@staticmethod` methods should have descriptive docstrings).

2. In the implementation of `get_query_dict`, adding comments or explanations where necessary would help clarify what's happening at that particular point.

Both changes are generally beneficial improvements but can likely remain optional without causing problems when implemented correctly.

However, due to the nature of this specific task requiring knowledge updated after September 2021 and beyond, there won't be any other meaningful corrections unless the user specifies any different data within the current date range (assuming they want to focus on an issue rather than making general advice about Django authentication).